DesktopEdge/MainWindow.xaml.cs

```diff
@@ -835,9 +835,18 @@ public partial class MainWindow : Window {
 				if ("installationupdate".Equals(evt.Message?.ToLower())) {
 					logger.Debug("Installation Update is available - {0}", evt.ZDEVersion);
 					IsUpdateAvailable = true;
-					MainMenu.ShowUpdateAvailable(evt.TimeRemaining, evt.InstallTime);

```

Comment:
was TimeRemaining bogus? or does it mean something else? should it be removed?

Reply:
It was utterly superfluous and i removed it from the json object the montior service sends, yes. It was replaced by a simple "now - installtime" calculation (which is already there):
```
var remaining = evt.InstallTime - DateTime.Now;
...
MainMenu.ShowUpdateAvailable(remaining.TotalSeconds, evt.InstallTime);
```